libs/sdk-ui-filters/src/DateFilter/index.ts

```diff
@@ -72,6 +72,8 @@ export {
     isUiRelativeDateFilterForm,
 } from "./interfaces/index.js";
 
+export { IFilterConfigurationProps } from "./DateFilterBody/types.js";
+
```

Comment:
I think we don't need the empty lines to separate the imports if they are on the same level.